ui/hooks/src/utils/error-handler.ts

```diff
@@ -4,6 +4,10 @@ import getSDK from '@akashaorg/awf-sdk';
 
 const logger = new Map<string, ILogger>();
 
+/**
+ * Utility to get a logger from a map of loggers
+ * @param name - name of logger
+ */
 export const getLogger = (name: string) => {
```

Comment:
is this still being used? This whole error-handler...?

Reply:
yes, the logError method is used to handle error in multiple hooks